problem_builder/public/js/instructor_tool.js

```diff
@@ -250,6 +250,88 @@ function InstructorToolBlock(runtime, element) {
         if (statusChanged) updateView();
     }
 
+    // Block types with answers we can export
+    var questionBlockTypes = ['pb-mcq', 'pb-rating', 'pb-answer'];
+
+    function getCourseBlocks() {
+        // Populate course tree options with course blocks
+        $.ajax({
+            type: 'GET',
+            url: $rootBlockId.data('course-blocks-api'),
+            data: {
+                course_id: $element.data('course-id'),
+                requested_fields: 'name,display_name,block_type,children',
+                student_view_data: questionBlockTypes.join(','),
+                all_blocks: true,
+                depth: 'all'
+            },
+            success: updateBlockOptions,
+            dataType: 'json'
+        });
+    }
+    function updateBlockOptions(data) {
+
+        var appendBlock = function(block) {
+            var blockId = block.id.split('+block@').pop(),
+                padding = Array(block.depth).join('&nbsp;'),
+                disabled = (block.enabled ? undefined : 'disabled'),
+                labelAttr,
+                label,
+                $option;
+
+            // Merge any fields exposed by student_view_data
+            block = _.extend(block, block['student_view_data']);
+
+            // Find the best label attribute available for the block
+            labelAttr = _.find(['question', 'name', 'display_name'],
+                               function(attr) {return block[attr]}),
```

Comment:
@pomegranited Nit: This would be even more readable when formatted like this:

```js
            labelAttr = _.find(
                ['question', 'name', 'display_name'],
                function(attr) { return block[attr] }
            ),
```

Or like this:

```js
            labelAttr = _.find(
                ['question', 'name', 'display_name'],
                function(attr) {
                    return block[attr]
                }
            ),
```